next.config.mjs

```diff
@@ -15,11 +15,13 @@ import { withSentryConfig } from "@sentry/nextjs";
 /** @type {import('next').NextConfig} */
 const config = {
   reactStrictMode: true,
+  output: "standalone",
```

Comment:
**question**: is that needed for the dockerfile?

Reply:
Yes, it is.